src/ddmd/root/filename.d

```diff
@@ -680,6 +679,8 @@ nothrow:
         }
         else version (Windows)
         {
+            import core.sys.windows.winbase: GetFullPathNameA;
```

Comment:
Why is something here using `GetFullPathNameA` instead of `GetFullPathNameW`? Doesn't that just invite the sort of the problems that this PR is supposed to fix?

Reply:
It does, and I wondered about this myself. The problem is that I'm not familiar with this code or what it does and it's used in a bunch of places.

Reply:
Isn't this what you want?
https://docs.microsoft.com/en-gb/cpp/c-runtime-library/reference/fullpath-wfullpath
_wfullpath

Reply:
@jmdavis @atilaneves ping

Reply:
@Laeeth No. All I did here was add a local import instead of depending on the module. Also, the Win32 API is preferable to Microsoft's C runtime.